reana_workflow_controller/consumer.py

```diff
@@ -67,12 +70,37 @@ def _update_workflow_status(workflow_uuid, status, logs):
     """Update workflow status in DB."""
     Workflow.update_workflow_status(Session, workflow_uuid,
```

Comment:
**Create issue after merge, not to amend on this PR:**

We could have a list of objects who subscribe to the workflow status and all of them implement `update_workflow_status`. Then we just simply iterate and call the method on each of them, so we simplify the implementation for GitLab which we have below.
```python
# Workflow is the DB model, so that subscribes the database to changes
subscribe_to_workflow_status_change = [Workflow, GitLab, Email]
for subscriber in subscribe_to_workflow_status_change:
    subscriber.update_workflow_status(workflow, status)
```